README.md

```diff
@@ -50,12 +50,12 @@ Only some types of objects can be retrieved from other types of objects:
 
 | `where=...`       | `cell`    | `cluster` | `dataset` | `gene`    | `organ`   | `protein` |
 | ----------------- | --------- | --------- | --------- | --------- | --------- | --------- |
-| [`select_cells()`](https://github.com/hubmapconsortium/hubmap-api-py-client/blob/main/examples/select_cells.md)                                                                                                              |           |           | ✓         | ✓         | ✓         | ✓         |
-| [`select_clusters()`](https://github.com/hubmapconsortium/hubmap-api-py-client/blob/main/examples/select_clusters.md)                                                                                                              |           |           |           | ✓ ✩       | ✩         | ✩         |
-| ~~`select_datasets()`~~                                                                                                              |           |           |           |           | ✶         |           |
-| [`select_genes()`](https://github.com/hubmapconsortium/hubmap-api-py-client/blob/main/examples/select_genes.md)                                                                                                              |           | ✓ ✩       |           |           | ✓ ✩       | ✩         |
-| [`select_organs()`](https://github.com/hubmapconsortium/hubmap-api-py-client/blob/main/examples/select_organs.md)                                                                                                              | ✓         | ✩         | ✶         | ✓ ✩       |           | ✩         |
-| ~~`select_proteins()`~~                                                                                                              |           | ✩         |           | ✩         | ✩         |           |
+| [`select_cells()`](https://github.com/hubmapconsortium/hubmap-api-py-client/blob/main/examples/select_cells.md)                                                                                                              | ✓         |           | ✓         | ✓         | ✓         | ✓         |
+| [`select_clusters()`](https://github.com/hubmapconsortium/hubmap-api-py-client/blob/main/examples/select_clusters.md)                                                                                                              |           | ✓         | ✓         | ✓ ✩       | ✩         | ✩         |
+| [`select_datasets()`](https://github.com/hubmapconsortium/hubmap-api-py-client/blob/main/examples/select_datasets.md)| ✓         | ✓         | ✓         |           |✶          | ✓         |
```

Comment:
Later you say:
```
'dataset': ['dataset', 'cell', 'cluster']
```
Should the last check for protein be removed?

Reply:
Yes, sorry about that.